src/extend/withPlugins.js

```diff
@@ -9,6 +9,11 @@ function _withPlugins(targetName, TargetComponent) { // eslint-disable-line no-u
   /** */
   function PluginHoc(props) {
     const pluginMap = useContext(PluginContext);
+
+    if (isEmpty(pluginMap)) {
+      return <TargetComponent {...props} />;
+    }
+
     const plugins = pluginMap[targetName];
 
     if (isEmpty(plugins)) {
```

Comment:
I wonder if we can delete this now?

Reply:
I think we definitely should keep that check -- lots of test cases break if it's removed after adding the `pluginMap` check above it. Presumably this is because the downstream checks of the properties of `plugins` don't cover all the bases.

I've submitted the PR as a single commit here:
https://github.com/ProjectMirador/mirador/pull/2620